src/RedialContext.js

```diff
@@ -188,7 +192,10 @@ export default class RedialContext extends Component {
 
     Promise.all(promises)
       .then(this.props.onCompleted)
-      .catch((err) => this.props.onError(err, bail() || 'other'));
+      .catch((err) => {
+        this.props.onError(err, bail() || 'other');
+        this.abort(true);
+      });
```

Comment:
Yes, that would probably be good! I'm all for this change!

Maybe it should be an option, since a scenario might be that the user reloads the page and would then get the correct resource?